hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -252,774 +253,774 @@
-          name: "RD",
+      { window: {
+          name: "prog_fifo",
+          items: "1",
+          validbits: "32",
+          byte-write: "false",
+          unusual: "false"
+          swaccess: "wo",
           desc: '''
-            When the read FIFO fills to this level, trigger an interrupt.
-            Default value is set such that interrupt does not trigger at reset.
+            Flash program FIFO.
+
+            The FIFO is 16 entries of 4B flash words. This FIFO can only be programmed
+            by software after a program operation has been initiated via the !!CONTROL register.
+            This ensures accidental programming of the program FIFO cannot lock up the system.
             '''
-          resval: "0xF"
         },
-      ]
-    }
+      },
 
-    { name: "FIFO_RST",
-      desc: "Reset for flash controller FIFOs",
-      swaccess: "rw",
-      hwaccess: "hro",
-      resval: "0",
-      fields: [
-        { bits: "0",
-          name: "EN",
+      { window: {
+          name: "rd_fifo",
+          items: "1",
+          validbits: "32",
+          byte-write: "false",
+          unusual: "false"
+          swaccess: "ro",
           desc: '''
-            Active high resets for both program and read FIFOs.  This is especially useful after the controller
-            encounters an error of some kind.
+            Flash read FIFO.
 
-            This bit will hold the FIFO in reset as long as it is set.
+            The FIFO is 16 entries of 4B flash words
             '''
-          resval: "0"
         },
-      ]
-    },
-
-    { window: {
-        name: "prog_fifo",
-        items: "1",
-        validbits: "32",
-        byte-write: "false",
-        unusual: "false"
-        swaccess: "wo",
-        desc: '''
-          Flash program FIFO.
-
-          The FIFO is 16 entries of 4B flash words. This FIFO can only be programmed
-          by software after a program operation has been initiated via the !!CONTROL register.
-          This ensures accidental programming of the program FIFO cannot lock up the system.
-          '''
       },
-    },
-
-    { window: {
-        name: "rd_fifo",
-        items: "1",
-        validbits: "32",
-        byte-write: "false",
-        unusual: "false"
-        swaccess: "ro",
-        desc: '''
-          Flash read FIFO.
-
-          The FIFO is 16 entries of 4B flash words
-          '''
-      },
-    },
-
-    { window: {
-        name: "prim_flash_cfg",
-        items: "21",
-        validbits: "32",
-        byte-write: "true",
-        // Setting the unusual flag avoids warning messages from regtool about
-        // the unusual number of items in the window.
-        unusual: "true"
-        swaccess: "rw",
-        desc: '''
-          Configuration window for prim_flash.
-
-          This window contains proprietary registers that is not exposed in open-source.
-          '''
-      },
-    },
-  ]
+    ],
+    prim: [
+//      { name: "DUMMY",
```

Comment:
`prim_flash_cfg` seems to have gone away, and we've gained a commented-out `DUMMY` register. (I'm amazed that an empty list of registers works: I never actually tried that!) Is that intentional?